draco/analysis/flagging.py

```diff
@@ -413,6 +416,47 @@ def process(self, data):
         return out
 
 
+class BadGainMask(task.SingleTask):
```

Comment:
Along the lines of the comment below, this might be better called `MaskBadGains`